libyaml_q_declaration.md

```diff
@@ -0,0 +1,135 @@
+
+
+# External Dependency Quality declaration libyaml
+
+This document is a declaration of software quality for the `libyaml` external dependency, based on the guidelines in [REP-2004](https://github.com/ros-infrastructure/rep/blob/rep-2004/rep-2004.rst).
+
+The [libyaml](https://github.com/yaml/libyaml) external dependency is a C library for parsing and emitting YAML. Maintained in the YAML Project Github organization together with other 115 repositories. First, a summary discussing how this library is qualified is presented, and then it will be listed how this library matches the standards defined for ROS packages.
+
+## Summary
+
+The `libyaml` meets the basic requirements for a software platform in terms of testing its basic functionality, providing a valid license for the code used and a public Github repository with the changes made to the code over time.
```

Comment:
can we link here the license? https://github.com/yaml/libyaml/blob/master/License

Reply:
Yes! Nice addition